### ISSUE TITLE
Object attributes do not get nullified properly

### ISSUE THREAD

**Issue by janjiss** [soveran/ohm#218]
Hello, I noticed that models `#load!` method does not actually nullify value if that value has been nullified on other instance or on other thread. It looks like that in the implementation, when you set something to nil, it removes the key from the db, but when we try to call `#load!`, since the key is not there, it does not update the attributes on the models instance.  

Here is a code snippet:
```ruby
  def test_ohm
    a = User.create(first_name: "Janis")
    b = User[a.id]
    b.update(first_name: nil)
    assert_equal nil, b.first_name
    assert_equal nil, a.load!.first_name
  end
```
The last assertion here fails

Thanks :)

**Comment by soveran** [comment]
@janjiss Great catch! I already have a fix and I'll release it later today. Thanks!

**Comment by soveran** [comment]
@janjiss I just released version 3.1.1, can you verify that it solves this issue? Again, thanks a lot for your help!

**Comment by janjiss** [comment]
@soveran Thank you for such a fast response! I can verify that the issue has been fixed
